src/cdk-experimental/menu/menu-item-trigger.ts

```diff
@@ -287,7 +287,6 @@ export class CdkMenuItemTrigger extends MenuTrigger implements OnDestroy {
 
   /** Determine and return where to position the opened menu relative to the menu item */
   private _getOverlayPositions(): ConnectedPosition[] {
-    // TODO: use a common positioning config from (possibly) cdk/overlay
```

Comment:
I couldn't really think of a way to create common configs that seemed useful. The only idea I really had is maybe we can create a function that takes your top preferred position and generates some mirrored fallbacks automatically. e.g.

```ts
getPositionWithFallbacks(
	{originX: 'start', originY: 'bottom', overlayX: 'start', overlayY: 'top'}
) === [
	{originX: 'start', originY: 'bottom', overlayX: 'start', overlayY: 'top'},
 	{originX: 'start', originY: 'top', overlayX: 'start', overlayY: 'bottom'},
	{originX: 'end', originY: 'bottom', overlayX: 'end', overlayY: 'top'},
	{originX: 'end', originY: 'top', overlayX: 'end', overlayY: 'bottom'},
]
```

Do you think this is worth it? Do you have other ideas?

Reply:
I had been imaging this as something roughly like 
```ts
const STANDARD_DROPDOWN_POSITIONS = [
  {originX: 'start', originY: 'bottom', overlayX: 'start', overlayY: 'top'},
  ...
];

const STANDARD_PICKER_POSITIONS = [...];
```
where the first would be the behavior that we use as the default for things with a list of items (menu, select, autocomplete), and picker would be for things like datepicker, color picker, that typically have more custom UI. The difference being the flexibility setting for the most part

Reply:
I think the problem with this is that there's not really a single standard set of positions. For horizontal menu bars and single button triggers the standard is to open the menu below the trigger. For sidebar menus and sub-menus the standard is to open the menu adjacent to the end of the trigger. We could save those both as different presets if we can come up with good names for them, but I'm having trouble coming up with good names that aren't a mouthful